src/passes/anti-hook/AntiHook.cpp

```diff
@@ -38,11 +38,31 @@ static const std::vector<PrologueInfoTy> ANTI_FRIDA_PROLOGUES = {
   )delim", 2}
 };
 
+// TODO: Ensure the following stub makes Frida fail.
```

Comment:
I checked the source code of Frida and I don't think this will be working for ARM/Thumb since Frida does not need a scratch register as for AArch64. It could be better to raise a warning or an error?